old/views/color.class.ts

```diff
@@ -1,8 +1,19 @@
 'use strict'
 
-const {ipcRenderer, remote, clipboard} = require('electron')
+const { ipcRenderer, remote, clipboard } = require('electron')
 
 class Color {
+
+  private alpha
+  private activeAlpha
+  private red
+  private green
+  private blue
+  private rgb
+  private rgba
+  private hex
+  private hsl
```

Comment:
TypeScript is a strongly-typed, therefore we try to create types for the variables we use...

Reply:
Don't take into consideration this file, it's an old typescript test.
See https://github.com/Toinane/colorpicker/pull/36/files/25878de6e13b3a0893126c3edb5579e66773aa91#diff-2fcaa4639dd6dbb9fcfb3cd1ae780d7fR26 instead 😄